### PR TITLE
Add React/Vue snippets for Chart Data Aggregation topics

### PR DESCRIPTION
**Pull request by dxBeardedBear**

### CODE REVIEW

File: concepts/05 Widgets/Chart/88 Data Aggregation/01 Enable Data Aggregation.md [hunk 1]
```diff
@@ -69,5 +69,74 @@ You can enable data aggregation for individual series, all series of a specific
...
-    
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ... >
+            <DxCommonSeriesSettings ...
+                :fullstackedbar="fullStackedBarSettings" <!-- Enables data aggregation for all Full-Stacked Bar series -->
+            >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/01 Enable Data Aggregation.md]:
```suggestion
                :fullstackedbar="fullStackedBarSettings"> <!-- Enables data aggregation for all Full-Stacked Bar series -->
```

File: concepts/05 Widgets/Chart/88 Data Aggregation/01 Enable Data Aggregation.md [hunk 2]
```diff
@@ -69,5 +69,74 @@ You can enable data aggregation for individual series, all series of a specific
...
+
+    const fullStackedBarSettings = { aggregation: { enabled: true } };
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Chart ... >
+                    <CommonSeriesSettings ...
+                        fullstackedbar={fullStackedBarSettings} {/* Enables data aggregation for all Full-Stacked Bar series */}
+                    >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/01 Enable Data Aggregation.md]:
```suggestion
                        fullstackedbar={fullStackedBarSettings}> {/* Enables data aggregation for all Full-Stacked Bar series */}
```

File: concepts/05 Widgets/Chart/88 Data Aggregation/05 Specify the Aggregation Interval.md [hunk 3]
```diff
@@ -46,6 +46,61 @@ Use the **argumentAxis**.[aggregationInterval](/api-reference/20%20Data%20Visual
...
 
+    ##### Vue
+
+        <!-- tab: App.vue -->
+        <template> 
+            <DxChart ... >
+                <DxArgumentAxis
+                    :aggregation-interval="100" <!-- A new interval every 100 units -->
+                    aggregation-interval="day"  <!-- A new interval every day -->
+                >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/05 Specify the Aggregation Interval.md]:
```suggestion
                    aggregation-interval="day"> <!-- A new interval every day -->
```

File: concepts/05 Widgets/Chart/88 Data Aggregation/05 Specify the Aggregation Interval.md [hunk 4]
```diff
@@ -46,6 +46,61 @@ Use the **argumentAxis**.[aggregationInterval](/api-reference/20%20Data%20Visual
...
+        } from 'devextreme-react/chart';
+
+        class App extends React.Component {
+            render() {
+                return (
+                    <Chart ... >
+                        <ArgumentAxis
+                            aggregationInterval={100} {/* A new interval every 100 units */}
+                            aggregationInterval="day"  {/* A new interval every day */}
+                        >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/05 Specify the Aggregation Interval.md]:
```suggestion
                            aggregationInterval="day"> {/* A new interval every day */}
```

File: concepts/05 Widgets/Chart/88 Data Aggregation/10 Choose the Aggregation Method.md [hunk 5]
```diff
@@ -65,6 +65,78 @@ The following code shows how to specify aggregation methods for each series indi
...
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ... >
+            <DxCommonSeriesSettings ... >
+                <DxAggregation :enabled="true"/>
+            </DxCommonSeriesSettings>
+            <DxSeries ...
+                type="line"
+            >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/10 Choose the Aggregation Method.md]:
```suggestion
                type="line">
```

File: concepts/05 Widgets/Chart/88 Data Aggregation/10 Choose the Aggregation Method.md [hunk 6]
```diff
@@ -65,6 +65,78 @@ The following code shows how to specify aggregation methods for each series indi
...
+                <DxAggregation method="min"/>
+            </DxSeries>
+            <DxSeries ...
+                type="bar"
+            >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/10 Choose the Aggregation Method.md]:
```suggestion
                type="bar">
```

File: concepts/05 Widgets/Chart/88 Data Aggregation/10 Choose the Aggregation Method.md [hunk 7]
```diff
@@ -65,6 +65,78 @@ The following code shows how to specify aggregation methods for each series indi
...
+    class App extends React.Component {
+        render() {
+            return (
+                <Chart ... >
+                    <CommonSeriesSettings ... >
+                        <Aggregation enabled={true} />
+                    </CommonSeriesSettings>
+                    <Series ...
+                        type="line"
+                    >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/10 Choose the Aggregation Method.md]:
```suggestion
                        type="line">
```

File: concepts/05 Widgets/Chart/88 Data Aggregation/10 Choose the Aggregation Method.md [hunk 8]
```diff
@@ -65,6 +65,78 @@ The following code shows how to specify aggregation methods for each series indi
...
+                        <Aggregation method="min" />
+                    </Series>
+                    <Series ...
+                        type="bar"
+                    >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/10 Choose the Aggregation Method.md]:
```suggestion
                        type="bar">
```

File: concepts/05 Widgets/Chart/88 Data Aggregation/20 Implement a Custom Aggregate Function.md [hunk 9]
```diff
@@ -78,5 +78,104 @@ In the following code, a custom aggregation function implements the <a href="htt
...
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ...
+            :data-source="data">
+            <DxSeries
+                argument-field="argument"
+                value-field="value"
+            >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/20 Implement a Custom Aggregate Function.md]:
```suggestion
                value-field="value">
```

File: concepts/05 Widgets/Chart/88 Data Aggregation/20 Implement a Custom Aggregate Function.md [hunk 10]
```diff
@@ -78,5 +78,104 @@ In the following code, a custom aggregation function implements the <a href="htt
...
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Chart ...
+                    dataSource={data}>
+                    <Series
+                        argumentField="argument"
+                        valueField="value"
+                    >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/88 Data Aggregation/20 Implement a Custom Aggregate Function.md]:
```suggestion
                        valueField="value">
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/Chart/88 Data Aggregation/01 Enable Data Aggregation.md (hunk 1 above)
Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/88 Data Aggregation/01 Enable Data Aggregation.md]: Fixed

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**